aodntools/timeseries_products/velocity_aggregated_timeseries.py

```diff
@@ -0,0 +1,316 @@
+import os
+import sys
+import netCDF4 as nc4
+import numpy as np
+import json
+from datetime import datetime
+import argparse
+
+import xarray as xr
+
+import aggregated_timeseries as TStools
+
+TEMPLATE_JSON = 'velocity_aggregated_timeseries_template.json'
+
+
+def sort_files(files_to_agg):
+    """
+    sort list of files according to deployment date
+    requires netcdf4 as nc4, dateutil.parser as parse
+    :param files_to_agg: List of files to sort
+    :return: sorted list of files
+    """
+
+    time_start = []
+    for file in files_to_agg:
+        with nc4.Dataset(file, 'r') as ds:
+            time_start.append(np.datetime64(ds.time_deployment_start))
+    tuples = sorted(zip(time_start, files_to_agg))
+    return [t[1] for t in tuples]
+
+
+def check_file(nc, site_code):
+    """
+    Return list of errors found in the file:
+    Variables of interest are present
+    TIME. LATITUDE, LONGITUDE,  is present
+    NOMINAL_DEPTH is not present as variable or attribute
+    file_version is not FV01
+    if LATITUDE or LONIGUTDE dimension has length >1
+
+    :param file: name of the netcdf file
+    :param nc: xarray dataset
+    :param VoI: string. Variable of Interest
+    :param site_code: code of the mooring site
+    :return: dictionary with the file name and list of failed tests
+    """
+
+    attributes = list(nc.attrs)
+    file_variables = list(nc.variables)
+    allowed_dimensions = ['TIME', 'LATITUDE', 'LONGITUDE', 'HEIGHT_ABOVE_SENSOR']
+    required_variables = ['UCUR', 'VCUR', 'WCUR']
+    error_list = []
+
```

Comment:
Wouldn't the allowed_dimensions and required_variables be better in a global variable (outside the functional scope?)?